build/build-image/cross/Dockerfile

```diff
@@ -15,12 +15,19 @@
 # This file creates a standard build environment for building cross
 # platform go binary for the architecture kubernetes cares about.
 
-FROM  golang:1.4
-MAINTAINER  Joe Beda <jbeda@google.com>
+FROM  golang:1.5.1
```

Comment:
@brendandburns - since we are upgrading testing infrastructure to go 1.5.2, can you please change it to 1.5.2 too?